spk/octoprint/Makefile

```diff
@@ -1,22 +1,25 @@
-SPK_NAME = octoprint
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 1
-SPK_ICON = src/octoprint.png
+SPK_NAME   = octoprint
+SPK_VERS   = $(shell date +%Y%m%d)-devel
```

Comment:
Depends on how long it takes until 1.3.0 is released, I guess. Using official releases is preferred because they are stable, but I don't mind that you use the develop branch if there's a good reason for it. But it isn't a reason to tack `-devel` onto the version :)

If you want (or need) to use the develop branch, you need to specify exactly which commit to use. That way, regardless of when the package is built or who does it, the resulting package is exactly the same as when you tested it.